pynsee/utils/save_df.py

```diff
@@ -44,15 +44,9 @@ def wrapper(*args, **kwargs):
             else:
                 file_name += ".pkl"            
                                     
-            if any([(a == 'update') & (kwargs[a] == True) for a in kwargs.keys()]):
-                update = True
-            else:
-                update = False  
+            update = any((a == 'update') and (kwargs[a] is True) for a in kwargs)
 
-            if any([(a == 'silent') & (kwargs[a] == True) for a in kwargs.keys()]):
-                silent = True
-            else:
-                silent = False    
+            silent = any((a == 'silent') and (kwargs[a] is True) for a in kwargs)
```

Comment:
If the above is correct, then this one should also be
```suggestion
            silent = kwargs.get("silence", False)
```